spec/check_command_allowlist_spec.rb

```diff
@@ -73,32 +73,50 @@ def setup_pipeline_file(path, content)
         YAML
       end
 
-      it "passes if command is in the pipeline.yml" do
-        expect(subject[:status]).to eq(0), "Output was: \n#{subject[:output]}"
-      end
+      shared_examples "correct behavior" do
```

Comment:
Ideally these shared examples would be used by all of the contexts (multiple commands, single commands, etc.) in this file but the test structure is a bit messy and it didn't seem too important to format this spec file in the perfect way for something so infrequently modified.